### PR TITLE
pages: add missing statis pages

### PR DESCRIPTION
**Pull request by zzacharo**

### CODE REVIEW

File: app_data/pages.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# CDS static pages
+- url: /about
+  title: About page
+  description: CDS about page
+  template: about.html
```
Comment (ntarocco) [inline app_data/pages.yaml:5]: as discussed, probably optional

File: app_data/pages.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# CDS static pages
+- url: /about
+  title: About page
```
Comment (ntarocco) [inline app_data/pages.yaml:3]:
```suggestion
  title: About
```

File: app_data/pages.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  description: CDS about page
```
Comment (ntarocco) [inline app_data/pages.yaml:4]:
```suggestion
  description: About page
```

File: app_data/pages.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+  template: about.html
+- url: /privacy-policy
+  title: Privacy policy page
```
Comment (ntarocco) [inline app_data/pages.yaml:7]:
```suggestion
  title: Privacy Notice
```

File: app_data/pages.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# CDS static pages
+- url: /about
+  title: About page
+  description: CDS about page
+  template: about.html
+- url: /privacy-policy
```
Comment (ntarocco) [inline app_data/pages.yaml:6]:
```suggestion
- url: /privacy-notice
```

File: app_data/pages.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+  title: Privacy policy page
+  description: CDS privacy policy page
```
Comment (ntarocco) [inline app_data/pages.yaml:8]:
```suggestion
  description: Privacy Notice page
```

File: app_data/pages.yaml [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+  template: privacy_policy.html
```
Comment (ntarocco) [inline app_data/pages.yaml:9]: remove the 2 pages below

File: templates/semantic-ui/invenio_app_rdm/footer.html [hunk 8]
```diff
@@ -25,7 +25,7 @@ <h2 class="ui small header">{{ _("About") }}</h2>
           </div>
           <div class="rdm-footer five wide tablet three wide computer eight wide mobile column">
             <h2 class="ui small header">{{ _("Help") }}</h2>
-            {% trans snow="https://cern.service-now.com/service-portal/function.do?name=CDS" %}
+            {% trans snow="https://cern.service-now.com/service-portal?id=service_element&name=CDS-Service" %}
```
Comment (ntarocco) [inline templates/semantic-ui/invenio_app_rdm/footer.html:28]: Above, can you change the URL and the text to:
`/privacy-notice`
`Privacy Notice`

**Review by ntarocco — APPROVED** [review]

**Review by kpsherva — APPROVED** [review]

**Pull request merged**